src/components/button/button.scss

```diff
@@ -19,9 +19,6 @@
 }
 
 button {
-    @include mixins.is-elevated-clickable();
```

Comment:
Shadows disappear when button is disabled. Maybe that's on purpose though?

<img width="168" alt="image" src="https://user-images.githubusercontent.com/154725/211846843-c753e51e-f69e-4a15-8dde-5688d28ad2d3.png">

Reply:
This was a side-effect but I think it's fine. A disabled button shouldn't have the "affordance" of being clicked. So no shadows would still work fine.

Reply:
I agree, it's nice now that I've gotten used to it 👍